src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -85,47 +88,47 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
             }
 
             [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational)]
-            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string eventCreatedAt)
+            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string timeCreated)
```

Comment:
I can take care of casing issue as part of [#885](https://github.com/Azure/azure-webjobs-sdk-script/issues/885 )